<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupPage</class>
 <widget class="QWidget" name="SetupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="text">
      <string>Tournament Setup</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_players">
       <property name="text">
        <string>Number of players:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="num_players_edit">
       <property name="placeholderText">
        <string>e.g. 8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_team_size">
       <property name="text">
        <string>Team size (x for x-vs-x):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="team_size_edit">
       <property name="text">
        <string>2</string>
       </property>
       <property name="placeholderText">
        <string>e.g. 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_courts">
       <property name="text">
        <string>Number of courts:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="court_count_edit">
       <property name="text">
        <string>1</string>
       </property>
       <property name="placeholderText">
        <string>e.g. 1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="text">
        <string>Next â€” Enter Names</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_button">
       <property name="text">
        <string>Load Tournament</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="example_button">
       <property name="text">
        <string>Load Example</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
